### PR TITLE
Update German translations

### PR DESCRIPTION
**Pull request by kahrpatrick**
### Context
Add missing German translations.

### Changes proposed in this pull request
This PR adds missing German translations and updates a few existing ones for better consistency.

### CODE REVIEW

File: app/views/spina/admin/shared/_navigation.html.erb [hunk 1]
```diff
@@ -33,14 +33,14 @@
...
           
           <div class="text-white font-semibold hidden md:block transform -translate-x-2 ease-in-out duration-300 absolute md:relative opacity-0 transition-all" data-navigation-target="label">
-            Settings
+            <%= t('spina.website.settings') %>
           </div>
         <% end %>
 
         <% nav.links do %>
-          <%= render Spina::MainNavigation::LinkComponent.new("General", spina.edit_admin_account_path, active: controller_name == "accounts") %>
+          <%= render Spina::MainNavigation::LinkComponent.new(t('spina.website.general'), spina.edit_admin_account_path, active: controller_name == "accounts") %>
```
Comment (Bramjetten) [inline app/views/spina/admin/shared/_navigation.html.erb:41]: Shouldn't this be spina.settings.general?

File: app/views/spina/admin/shared/_navigation.html.erb [hunk 2]
```diff
@@ -33,14 +33,14 @@
...
           <%= render Spina::MainNavigation::LinkComponent.new(t('spina.website.theme'), spina.edit_admin_theme_path, active: controller_name == "theme") %>
-          <%= render Spina::MainNavigation::LinkComponent.new("Users", spina.admin_users_path, active: controller_name == "users") %>
+          <%= render Spina::MainNavigation::LinkComponent.new(t('spina.website.users'), spina.admin_users_path, active: controller_name == "users") %>
```
Comment (Bramjetten) [inline app/views/spina/admin/shared/_navigation.html.erb:43]: And this spina.settings.users?

File: config/locales/en.yml [hunk 3]
```diff
@@ -108,6 +108,7 @@ en:
       delete_confirmation_html: Are you sure? This attachment <span class='font-semibold'>might</span> be in use on a page.
       insert: Insert document
       insert_multiple: Insert documents
+      upload: Upload
```
Comment (Bramjetten) [inline config/locales/en.yml:111]: Can you change this to Upload files?

File: app/views/spina/admin/shared/_navigation.html.erb (hunk 1 above)
Comment (kahrpatrick) [reply app/views/spina/admin/shared/_navigation.html.erb:41]: I agree, that makes more sense.

**Review by Bramjetten — APPROVED** [review]

**Pull request merged**